src/requestor.js

```diff
@@ -3,7 +3,23 @@
 class LMRTFYRequestor extends FormApplication {
     constructor(...args) {
         super(...args)
-        game.users.apps.push(this)
+        game.users.apps.push(this);
```

Comment:
Is this required? I haven't seen any other applications do this before.

Reply:
I didn't add that. Seems like I just added the semicolon behind the line when I was on autopilot. I can remove the semicolon if needed, but the line of code itself was there to begin with.

Reply:
Hm. You don't need to revert the change, though I do wonder why that is there.